### PR TITLE
markdown mostly

### PR DESCRIPTION
**Pull request by jsoref**
The github editor and my text editor don't get along well, so you'll want to select individual fixes.  

Plus there are a few questions embedded.

### CODE REVIEW

**Comment by kamrik** [comment]
For some reason your editor uses "\xc2\xa0" instead of some spaces. 
This is a utf-8 representation of the Unicode non breaking space U+00A0.
What do you use for an editor? :)

File: 2014-07-09-tools-release.md [hunk 1]
```diff
@@ -29,57 +29,67 @@ Other changes include:
...
+* [CB-6943](https://issues.apache.org/jira/browse/CB-6943) Path can include the : if it is absolute, only test for `http`. Added tests
+* Show full stack for `CordovaError` in verbose mode
 * [CB-6024](https://issues.apache.org/jira/browse/CB-6024) Use nopt instead of optimist in cli
-* Add --usenpm flag to activate npm based lazy_load
+* Add `--usenpm` flag to activate npm based `lazy_load`‎
 * [CB-6767](https://issues.apache.org/jira/browse/CB-6767) Allow `cordova` to be replaceable in error messages
-* Add --noregstry flag for disabling plugin lookup in the registry
+* Add `--noregstry` flag for disabling plugin lookup in the registry‎
 
+••• noregstry or noregistry ‎•••?
```
Comment (kamrik) [inline 2014-07-09-tools-release.md:41]: that's a typo, it's with 'i' in the code

File: 2014-07-09-tools-release.md [hunk 2]
```diff
@@ -29,57 +29,67 @@ Other changes include:
...
-* support for shrinkwrap flag
+* [CB-6854](https://issues.apache.org/jira/browse/CB-6854) Strip BOM when adding `cordova.define()` to js-modules
+* Use `PluginInfo` in `plugman/install.js`‎
+* [CB-6772](https://issues.apache.org/jira/browse/CB-6772) Provide a default for `AndroidLaunchMode`‎
+* [CB-6711](https://issues.apache.org/jira/browse/CB-6711) Use `parseProjectFile` when working with XCode projects.
+* Start using `PluginInfo` object in `plugman/install.js`‎
+* [CB-6709](https://issues.apache.org/jira/browse/CB-6709) Remove `merges/` folder for default apps
+* support for `‎shrinkwrap` flag
+
+••• --shrinkwrap? •••
```
Comment (kamrik) [inline 2014-07-09-tools-release.md:84]: this is for the save/restore plugin, if you want plugin versions to be saved as well as the names. Will clarify.

File: 2014-07-09-tools-release.md [hunk 3]
```diff
@@ -29,57 +29,67 @@ Other changes include:
...
 * [CB-6895](https://issues.apache.org/jira/browse/CB-6895) Add more config properties into manifest
-* Allow "cordova platform add platform@version"
+* Allow `cordova platform add platform@version`‎
 * Add util func for chaining promises
-* removing doWrap from prepare
+* removing `doWrap` from `prepare`‎
 * adding prepare-browserify
 * adding options to prepare
+
+••• this isn't remotely useful •••
```
Comment (kamrik) [inline 2014-07-09-tools-release.md:65]: will remove.

**Comment by jsoref** [comment]
Oh gee...

Well, I guess that's technically plain text...

I'm using my phone and the email client, I wonder if the notes application would have been different (most applications are based around html editors these days)

File: 2014-07-09-tools-release.md (hunk 2 above)
Comment (jsoref) [inline 2014-07-09-tools-release.md:84]: I just want the hyphens included assuming that's how it's used (if it's one, then one...)

**Comment by kamrik** [comment]
Ok, I'll strip the unicode chars (that's a search-n-replace for me) and add merge the rest of your changes. Thanks!

**Pull request closed without merging**